src/blocks/Footer.vue

```diff
@@ -5,26 +5,26 @@
       <div class="_display-flex _flex-direction-row container _align-items-center _justify-content-space-between">
         <i-row center class="linksRow _margin-0 _margin-md-top-1 _flex-nowrap _text-nowrap">
           <div class="_padding-1-2 _padding-md-x-1 _padding-left-0">
-            <a target="_blank" class="footer-link" href="https://zksync.io/faq/intro.html">Docs</a>
+            <a target="_blank" class="footer-link" href="https://developers.rsk.co/rif/aggregation/">Docs</a>
           </div>
           <div class="_padding-1-2 _padding-md-x-1">
-            <a target="_blank" class="footer-link" href="https://zksync.io/legal/terms.html">Terms</a>
+            <a target="_blank" class="footer-link" href="https://www.rsk.co/terms-conditions">Terms</a>
           </div>
           <div class="_padding-1-2 _padding-md-x-1">
-            <a target="_blank" class="footer-link" href="https://zksync.io/contact.html">Contact</a>
+            <a target="_blank" class="footer-link" href="https://www.iovlabs.org/contact.html">Contact</a>
           </div>
           <div class="_padding-1-2 _padding-md-x-1">
             <span class="dividerLine">|</span>
           </div>
           <div class="_padding-1-2 _padding-md-x-1">
-            <a target="_blank" class="footer-link" :href="blockExplorerLink">zkScan</a>
+            <a target="_blank" class="footer-link" :href="blockExplorerLink">Explorer</a>
           </div>
         </i-row>
         <i-row center class="_text-nowrap _align-items-center _margin-0">
           <div class="_display-flex _align-items-center _padding-1-2 _padding-md-top-2 _padding-md-bottom-1 _padding-right-1 _hidden-sm-and-down">
             <block-system-info />
             <span class="_padding-x-1 dividerLine">|</span>
-            <a href="https://uptime.com/s/zksync" class="uptime-link" target="_blank">uptime</a>
+            <a href="https://uptime.com/s/rif-aggregation" class="uptime-link" target="_blank">uptime</a>
```

Comment:
For the uptime, I don't know how it works. Maybe we need to ask devops if we have some similar services to be used.

Reply:
Yes, I will remove it for now, like in the explorer, until we have something about that